autoexec.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Love the filename 💾 

Maybe `#!/usr/bin/env bash` to match other bash scripts here

Reply:
Done